ui/.env.production.example

```
@@ -44,4 +44,10 @@ REACT_APP_NETSTATED_URL=https://netstated-d7bbec1ed55b.herokuapp.com/exec
 
 # CMS and Translations
 STRAPI_API_TOKEN=<STRAPI_API_TOKEN>
-STRAPI_API_URL=https://cms.lantern.io/api
\ No newline at end of file
+STRAPI_API_URL=https://cms.lantern.io/api
+# where the freeze watchdog beacons crash/freeze reports (egress /freeze endpoint).
+# Leave empty to disable beaconing: reports then stay in the console and on
+# window.__unboundedWatchdog, which is where every freeze died before this existed.
+# NB this sends the *embedding site's* URL, since that is the field that says which page
+# froze — see the header comment in ui/src/utils/freezeWatchdog.ts.
+REACT_APP_FREEZE_BEACON_URL=https://unbounded.iantem.io/freeze
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REACT_APP_FREEZE_BEACON_URL key should go before the STRAPI_API_TOKEN key

(UnorderedKey)